dw-select.js

```diff
@@ -459,7 +507,7 @@ export class DwSelect extends DwFormElement(LitElement) {
     this.label = '';
     this.dialogTitle = '';
     this.hAlign = 'left';
-    this.vAlign = 'top';
+    this.vAlign = 'bottom';

```

Comment:
@nirmalbaldaniya-dream यहा पर default value change करनेकी जरूरत कयु हे?

Reply:
Previous value was based on anchor element. While new value is for dialog itself. When it's top dialog would be opened on top side of anchor element & when it's bottom, it would be open on bottom side of anchor element.

We have introduced "showTrigger" property to hide/show anchor element.

Reply:
@nirmalbaldaniya-dream इसका मतलब यह की यह changes backward compatible नहीं हे, right ?